app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
```

Comment:
Concider separating this into lines completely, if you already started doing this 
```suggestion
    def __init__(self, 
                 name: str,
                 appetite: int,
                 is_hungry: bool = True) -> None:
```

Reply:
Fix the same for other inits